diff --git a/dist/declarations/src/index.d.ts b/dist/declarations/src/index.d.ts
index df6e0869a794df65f3754f8f7f4e5c36c0750860..fc8087573eb9001d8a8166d13933ed8dd30ec098 100644
--- a/dist/declarations/src/index.d.ts
+++ b/dist/declarations/src/index.d.ts
@@ -1,5 +1,5 @@
 declare const DEPENDENCY_TYPES: readonly ["dependencies", "devDependencies", "peerDependencies", "optionalDependencies"];
-export type VersionType = "major" | "minor" | "patch" | "none";
+export type VersionType = "major" | "minor" | "minor-isolated" | "patch" | "patch-isolated" | "none";
 export type DependencyType = typeof DEPENDENCY_TYPES[number];
 export type AccessType = "public" | "restricted";
 export type Release = {
