diff --git a/dist/kdbxweb.js b/dist/kdbxweb.js
index 7d330745e3ebc4c99c09a573824ae3bb08583565..33b6b8fc04d76e186d0cef951937a3137a36603f 100644
--- a/dist/kdbxweb.js
+++ b/dist/kdbxweb.js
@@ -5342,16 +5342,8 @@ function createDOMParser() {
         return new __webpack_require__.g.DOMParser();
     }
     const parserArg = {
-        errorHandler: {
-            warning: (e) => {
-                throw e;
-            },
-            error: (e) => {
-                throw e;
-            },
-            fatalError: (e) => {
-                throw e;
-            }
+        onError: (_level, message) => {
+            throw new Error(message);
         }
     };
     /* eslint-disable @typescript-eslint/no-var-requires,@typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-call */
diff --git a/lib/utils/xml-utils.ts b/lib/utils/xml-utils.ts
index 2d208bd92ef07d5ec30daf26a83f1c72dc9f1c38..7038edbf804c0365f2118065842b938faf66e728 100644
--- a/lib/utils/xml-utils.ts
+++ b/lib/utils/xml-utils.ts
@@ -26,16 +26,8 @@ function createDOMParser() {
     }
 
     const parserArg = {
-        errorHandler: {
-            warning: (e: Error) => {
-                throw e;
-            },
-            error: (e: Error) => {
-                throw e;
-            },
-            fatalError: (e: Error) => {
-                throw e;
-            }
+        onError: (_level: string, message: string) => {
+            throw new Error(message);
         }
     };
 
